bikeshed/boilerplate/i18n/status-ED.include

```diff
@@ -11,10 +11,6 @@
 	(<a href="mailto:www-international-request@w3.org">subscribe</a>,
 	<a href="https://lists.w3.org/Archives/Public/www-international/">archives</a>).
 
-<p>
-	This document was produced by the <a href="http://www.w3.org/Style/CSS/members">CSS Working Group</a>
```

Comment:
The first part of this sentence is still part of the other boilerplates.

Reply:
I removed this sentence since it's in the i18n boilerplate, and the Internationalization Working Group was already mentioned in the paragraph above the sentence.

Reply:
ah missed that. Sorry.